plugins/source_validate_test.go

```diff
@@ -79,21 +79,27 @@ func TestSourcePlugin_listAndValidateAllResources(t *testing.T) {
 		},
 		{
 			name:                "should return an error if child table is without its parent",
-			plugin:              SourcePlugin{tables: []*schema.Table{{Name: "table 1", Parent: &schema.Table{Name: "table 2"}}, {Name: "table 2"}}},
-			configurationTables: []string{"table 1"},
+			plugin:              SourcePlugin{tables: []*schema.Table{{Name: "main_table", Relations: []*schema.Table{{Name: "sub_table", Parent: &schema.Table{Name: "main_table"}}}}}},
+			configurationTables: []string{"sub_table"},
 			wantErr:             true,
 		},
+		{
+			name:                "should return both tables if both child and parent are specified",
```

Comment:
Yeah hmm, actually I think this should either:
 - return an error if a child table is specified, or
 - remove the child table from the list as long as the parent is specified, since it will get fetched anyway

I don't think we want to return the child table, since if we try to resolve it on its own the parent value will be `nil`.

Reply:
Removed that test case as this seems like another issue.

Reply:
Yeah, you're right, looks like this is a separate issue. But we should address it, otherwise we'll probably get a panic with some configurations.

Reply:
Created an issue to track it: https://github.com/cloudquery/plugin-sdk/issues/340

Reply:
I never said we shouldn't address it.

Reply:
Neither did I try to imply that, sorry if it came across that way :bow: